recipes/xsdata/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "xsdata" %}
+{% set version = "20.11.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: ab484bc3384b9f4e278f245724a3fbe0c868e5551b42b3ba13ead80d498866e1
+
+build:
```

Comment:
It looks like this package has a python entry point so you should add that:

https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#python-entry-points

and then add a test of it to the `commands`:

https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#test-commands

Reply:
Thank you so much for your help!